src/frontend/src/components/common/Header/Header.tsx

```diff
@@ -44,15 +44,15 @@ function Header({ user }: Props) {
       setAccessToken(keycloakToken.access_token);
       navigate(routes.PPQ_LANDING_PAGE);
     } else return;
-  }, [code, keycloakToken, authCodeError]);
+  }, [code, keycloakToken, authCodeError, win.localStorage, navigate]);
```

Comment:
did you test this code after adding two dependencies? is everything working as expected?

Reply:
My apologies, this was an autofix from eslint. I have removed them as the navigate dependency in particular was a breaking change. Thanks for catching this.